go/test/endtoend/backup/vtbackup/backup_only_test.go

```diff
@@ -367,3 +381,14 @@ func verifyDisableEnableRedoLogs(ctx context.Context, t *testing.T, mysqlSocket
 		}
 	}
 }
+
+// insert should not work for any of the replicas and primary
+func getVTExecVersion(binaryName string) (int, error) {
+	vtTabletVersion := 0
+	vtTabletVersion, err := cluster.GetMajorVersion(binaryName)
+	if err != nil {
+		return 0, err
+	}
+	log.Infof("cluster.VtTabletMajorVersion: %d", vtTabletVersion)
```

Comment:
I think we should get rid of this function, but if not... it's not (always) vttablet involved here.

Reply:
I didn't understand the comment here... we always have vttablet here right...

Reply:
resolved